src/languages/lsl.js

```diff
@@ -0,0 +1,90 @@
+/*
+Language: Linden Scripting Language
+Description: The Linden Scripting Language is used in Second Life by Linden Labs.
+Author: Builder's Brewery <buildersbrewery@gmail.com>
+Category: scripting
+*/
+
+function(hljs) {
+
+    var LSL_STRING_ESCAPE_CHARS = {
+        className: 'subst',
+        begin: '\\[\\\\nt"\\]'
```

Comment:
@isagalaev That leaves us with `\\`, `\"`, `\n` and `\t` as the only escape sequences for double-quoted strings in LSL. How to change the regex then?